build/cockroach-template.rb

```diff
@@ -16,29 +16,22 @@ def install
     ENV["GOPATH"] = buildpath
     ENV["GOHOME"] = buildpath
 
-    files = Dir.glob("*")
+    # TODO(peter): It would be nicer to do something like:
+    #     git ls-tree --name-only HEAD
+    files = Dir.glob("*") + Dir.glob(".{git*,dockerignore,editorconfig}")
```

Comment:
How about `Dir.glob("*") + Dir.glob(".[a-z]*")` (assuming all you really care about is getting everything but `.` and `..`)?